### PR TITLE
Add checkpoint for ownership restore:

### PR DESCRIPTION
**Pull request by yafu-1**
1.It only contained selinux label restore before;
2.Add checkpoint for ownership restore since bug#547546 fixed;

Signed-off-by: Yan Fu <yafu@redhat.com>

### CODE REVIEW

**Comment by yafu-1** [comment]
Test result:
# avocado run svirt_start_destroy --vt-type libvirt
/root/.libvirt-ci-venv-ci-runtest-c9mAOr/lib/python3.6/site-packages/avocado_framework-77.0-py3.6.egg/avocado/plugins/run.py:287: FutureWarning: The following arguments will be changed to boolean soon: sysinfo, output-check, failfast, keep-tmp and ignore-missing-references.
  FutureWarning)
JOB ID     : c420cc6d4d1776a3bebf635538965aa006ca96b8
JOB LOG    : /root/avocado/job-results/job-2020-07-02T06.43-c420cc6/job.log
 (01/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_virt_content.relabel_yes.st_dynamic.default_model.without_qemu_conf: PASS (34.47 s)
 (02/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_virt_content.relabel_yes.st_static.default_model.without_qemu_conf: PASS (35.39 s)
 (03/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_static.default_model.with_qemu_conf.unconfined: PASS (30.04 s)
 (04/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_static.default_model.without_qemu_conf: PASS (35.36 s)
 (05/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_none.multi_model.with_qemu_conf.unconfined: PASS (32.86 s)
 (06/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_none.multi_model.without_qemu_conf: PASS (27.30 s)
 (07/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_none.default_model.with_qemu_conf.unconfined: PASS (30.35 s)
 (08/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_no.st_none.default_model.without_qemu_conf: PASS (28.22 s)
 (09/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_yes.st_dynamic.no_model.with_qemu_conf.security_driver_none: PASS (29.78 s)
 (10/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_yes.st_dynamic.no_model.without_qemu_conf: PASS (35.95 s)
 (11/44) type_specific.io-github-autotest-libvirt.svirt_start_destroy.positive_test.off_destroy.d_svirt_img_s0.no_baselabel.relabel_yes.st_dynamic.default_model.without_qemu_conf: PASS (34.19 s)
...
RESULTS    : PASS 44 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1680.00 s

**Comment by yafu-1** [comment]
@chloerh @Yingshun 
Could you help to review the pr please?
Thanks.

File: libvirt/tests/src/svirt/svirt_start_destroy.py [hunk 1]
```diff
@@ -181,25 +184,44 @@ def _check_label_equal(label1, label2):
...
+            f = os.open(filename, 0)
+            stat_re = os.fstat(f)
+            ownership_of_disk = "%s:%s" % (stat_re.st_uid, stat_re.st_gid)
+            logging.debug("The ownership of disk after guest starting is:\n")
+            logging.debug(ownership_of_disk)
+            logging.debug("The ownership of disk before guest starting is:\n")
+            logging.debug(backup_ownership_of_disks[filename])
+            if not (sec_relabel == "no" and sec_type == 'none'):
+                if not libvirt_version.version_compare(5, 6, 0):
+                    if (not img_label_after == img_label):
```
Comment (Yingshun) [inline libvirt/tests/src/svirt/svirt_start_destroy.py:209]: if img_label_after != img_label:

File: libvirt/tests/src/svirt/svirt_start_destroy.py [hunk 2]
```diff
@@ -6,6 +6,9 @@
 from virttest import utils_config
 from virttest import utils_libvirtd
 from virttest.libvirt_xml.vm_xml import VMXML
+from virttest import utils_test
+
+from provider import libvirt_version
```
[the remark this answers is not in the document]
Comment (yafu-1) [reply libvirt/tests/src/svirt/svirt_start_destroy.py]: Updated.

File: libvirt/tests/src/svirt/svirt_start_destroy.py [hunk 3]
```diff
@@ -181,25 +184,44 @@ def _check_label_equal(label1, label2):
...
-
-                err_msg = "Label of disk is not restored in VM shuting down.\n"
-                err_msg += "Detail: img_label_after=%s, " % img_label_after
-                err_msg += "img_label_before=%s.\n" % img_label
-                err_msg += "More info in https://bugzilla.redhat.com/show_bug"
-                err_msg += ".cgi?id=547546"
-                test.fail(err_msg)
+            filename = list(disks.values())[0]['source']
+            img_label_after = utils_selinux.get_context_of_file(filename)
+            f = os.open(filename, 0)
```
[the remark this answers is not in the document]
Comment (yafu-1) [reply libvirt/tests/src/svirt/svirt_start_destroy.py]: Replaced with 'os.stat()' which does not need to open file.

File: libvirt/tests/src/svirt/svirt_start_destroy.py (hunk 1 above)
Comment (yafu-1) [reply libvirt/tests/src/svirt/svirt_start_destroy.py]: Updated.

**Pull request merged**